### ISSUE TITLE
Extension activation broken on VS Code Insiders

### ISSUE THREAD

**Issue by nonameolsson** [letmaik/vscode-git-tree-compare#23]
Great extension! Recently, during last week, the extension stopped showing up. And the commands doesn't work. This is the error message I get.
![screen shot 2017-08-29 at 10 16 31](https://user-images.githubusercontent.com/1493018/29828723-3117b908-8ca3-11e7-9c2e-a6170e2e8f75.png)

I use VS Code: Version 1.16.0-insider (1.16.0-insider)

**Comment by letmaik** [comment]
I'll have a look, thanks for the report!

**Comment by letmaik** [comment]
I found the issue. The view of the extension is only displayed when the workspace is a git repo, and for that I rely on an undocumented feature of the internal git extension of VS Code which has changed in the latest Insiders build due to the introduction of multiple workspace folders.

Technical details:
The part I rely on is the context key `gitState` and I include that check in the `package.json` as `gitState != norepo`. In the latest Insiders build this context key is not there anymore, I assume due to the change to multiple workspace folders. The internal git extension now has a key `gitOpenRepositoryCount` instead. I tried that but it didn't work and it seems as if context from one extension cannot be accessed from another extension anymore. I'll have to verify that and may have to find a work-around, or if all else fails always display the view, no matter if it's a git repo or not, which would be a shame.

My current recommendation would be to use the non-insiders version.
